### DISCUSSION TITLE
Main navigational keys not working on crates.io

### DISCUSSION THREAD

**Discussion by grtcdr** [qutebrowser/qutebrowser#6863] [Q&A] [answered]
I didn't want to file a bug report, because this might be an issue on crates.io's end, but the main navigational keys (_hjkl_) don't seem to be working. I can link my configuration but I don't think that's the issue, as they work fine on every other website I visit.

**Comment by mschilli87** [comment, marked as answer]
@grtcdr
> [T]he main navigational keys (hjkl) don't seem to be working.

What do you expect them to do?
For me, `j` scrolls down and `k` scrolls up, at least when I don't focus on the search box.

**Reply by grtcdr** [reply]
Well, I expect them to scroll.

**Reply by mschilli87** [reply]
As I said, they do for me. Can you reproduce this using `--temp-basedir`? Also, try hitting TAB before you start scrolling to makesure you are not focussed on the search field.

**Reply by grtcdr** [reply]
Your tab tip worked :) Thanks a lot :heart:
